portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Ibiyemi!</h1>");
```

Comment:
Add a line to your website that fetches this data and inserts it somewhere. Even if it's just in an alert somewhere.

Reply:
The tutorial said to stop here and make a pull request, I was planning to make the version that inserts this into the body PR #14.